blocks/footer/footer.js

```diff
@@ -64,6 +65,28 @@ function createMenuAccordion(footer) {
   });
 }
 
+/**
+ * @param fragment
+ * @param footerPath
+ * @param locale
+ * Creates the modal trigger button in footer
+ */
+function createModalButton(fragment, footerPath, locale) {
+  const footerModalPath = getMetadata('footer-modal-path') || '/forms/modals/modal';
+  const btn = p(
+    { class: 'button-container quote-wrapper' },
+    a({ href: footerModalPath, class: 'quote-button button primary', 'aria-label': 'Request a Free Quote' }, 'Request a Free Quote'),
```

Comment:
Are we not handling the static text for placeholders in forms for other locales ?